[DOC] - Change log for 6.1.0-beta 2. (#77)
Iterable/swift-sdk: CHANGELOG.md

All notable changes to this project will be documented in this file.
This project adheres to [Semantic Versioning](http://semver.org/).

## 6.1.0-beta2
#### Added

> - Support for `action://your-custom-action-name` url scheme for calling custom actions and reserved `itbl://sdk-custom-action` scheme for SDK internal actions. 

Reviewer: ```suggestion
- Support for `action://your-custom-action-name` URL scheme for calling custom actions 
- Support for reserved `itbl://sdk-custom-action` scheme for SDK internal actions. 
```

> 	- For example, if you want a custom action called `buyCoffee` to be called via `IterableCustomActionDelegate` when user taps on the 'Buy' button in your in-app, you have to set the href of the button as `action://buyCoffee`.

Reviewer: ```suggestion
	- For example, to have `IterableCustomActionDelegate` call a custom `buyCoffee` action when a user taps on an in-app message's **Buy** button, set the href of the button to `action://buyCoffee`.
```

How is the button URL labeled when setting up the in-app? Is it **href** or is it **URL**? Would probably change this sentence to match if necessary.

I'd also put this underneath the appropriate bullet above (since I separated them)

Author: It is like any other regular html button so it is **href**.

> 	- Url scheme `itbl://sdk-custom-action` is reserved for internal SDK actions. Do not use `itbl://` urls for your custom actions. For example, in the future you will be set the href for your 'Delete' button to `itbl://delete` for the SDK to delete your in-app when the 'Delete' button is tapped.

Reviewer: ```suggestion
	- URL scheme `itbl://sdk-custom-action` is reserved for internal SDK actions. Do not use it for custom actions. 
	- For example, future versions of the SDK may allow buttons to call href `itbl://delete` to delete an in-app message.
```

- As above, should this be "href" or "URL"?
- Move this under the appropriate bullet above (since I separated them)
